src/main/java/games/strategy/engine/chat/Chat.java

```diff
@@ -49,14 +49,14 @@
   private final ChatIgnoreList ignoreList = new ChatIgnoreList();
   private final HashMap<INode, LinkedHashSet<String>> notesMap = new HashMap<>();
   private static final String TAG_MODERATOR = "[Mod]";
-  private final CHAT_SOUND_PROFILE chatSoundProfile;
+  private final ChatSoundProfile chatSoundProfile;
 
-  public enum CHAT_SOUND_PROFILE {
+  public enum ChatSoundProfile {
```

Comment:
Just a suggestion...  It seems unnecessary to prefix the enum type with `Chat` since it is embedded within the `Chat` class.  Would `Chat.SoundProfile` read better than `Chat.ChatSoundProfile`?

Reply:
As an inner enum, that would make sense. Though, I'm not 100% sure if it should be an inner enum.